gradio/inputs.py

```diff
@@ -50,6 +52,13 @@ def __init__(self, lines=1, placeholder=None, default=None, numeric=False, type=
             self.type = "number"
         else:
             self.type = type
+        if default is None:
```

Comment:
I think all the loading test data should be in a separate function instead of merged within the init function